tests/src/availability/concourse_test.go

```diff
@@ -0,0 +1,122 @@
+package availability_test
+
+import (
+	"crypto/tls"
+	"net"
+	"net/http"
+	"sort"
+	"time"
+
+	"github.com/concourse/atc"
+	"github.com/concourse/go-concourse/concourse"
+
+	. "github.com/onsi/gomega"
+)
+
+const (
+	pipelineName = "create-bosh-cloudfoundry"
+	jobName      = "cf-deploy"
+	resourceName = "pipeline-trigger"
+)
+
+type byStartTime []atc.Build
+
+func (a byStartTime) Len() int           { return len(a) }
+func (a byStartTime) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+func (a byStartTime) Less(i, j int) bool { return a[i].StartTime > a[j].StartTime }
+
+type basicAuthTransport struct {
+	username string
+	password string
+	base     http.RoundTripper
+}
+
+func getConcourseAtcUrl() string {
+	return getConfigFromEnvironment("CONCOURSE_ATC_URL")
+}
+func getConcourseUserName() string {
+	return getConfigFromEnvironment("CONCOURSE_ATC_USERNAME")
+}
+func getConcoursePassword() string {
+	return getConfigFromEnvironment("CONCOURSE_ATC_PASSWORD")
+}
+func getSkipSSLValidation() bool {
+	return getConfigFromEnvironment("SKIP_SSL_VALIDATION") == "true"
+}
+
+func filterAndSortBuildsByJobName(builds []atc.Build, jobName string) []atc.Build {
+	var filteredBuilds []atc.Build
```

Comment:
just a nitpick, but it should be more like filterByNameAndSortByTime, or more simply SortByTime(builds, jobname) - jobname parameter indicates that you want to sort this job by time